cucumber/features/steps/query.rb

```diff
@@ -25,9 +25,18 @@ def newlines_in_queries_supported?
 end
 
 Then(/^I query for the Silly Zero button by mark using the title$/) do
-  elements = query({marked: "Alpha"})
-  expect(elements.count).to be == 1
-  expect(elements[0]["id"]).to be == "alpha button"
+  elements = query({marked: "Zero"})
+  expected_id = "zero button"
+  # `Button` have `StaticText` element as a children in order to visualize its text content starting from iOS 13.
+  # Properties `label` of these elements are equals between each other.
+  if ios_gte?("13.0")
```

Comment:
Are `marked` queries are broken in general?  Or has something changed about buttons?

Do we have change something here:

https://github.com/calabash/DeviceAgent.iOS/blob/develop/Server/AutomationActions/Query/Specifiers/QuerySpecifierByMark.m

Reply:
Resolved